### ISSUE TITLE
Error "Failed to notifiy project evaluation listener" in Gradle 5

### ISSUE THREAD

**Issue by agpfrick** [evant/gradle-retrolambda#273]
Hello.
I tried building simple hello world project in Gradle 5 but it has this error:
```
* What went wrong:
A problem occurred configuring root project 'myjava'.
> Failed to notify project evaluation listener.
   > org.gradle.api.tasks.SourceSetOutput.getClassesDir()Ljava/io/File;
```
If I use gradle 4.x, it compiles fine. Any idea to fix this? Thank you.

**Comment by DanySK** [comment]
Same error here, affecting Protelis/Protelis#195

**Comment by evant** [comment]
Released version 3.7.1 which should be compatible with gradle 5+
